api/queries_pr.go

```diff
@@ -780,6 +780,25 @@ func PullRequestClose(client *Client, repo ghrepo.Interface, pr PullRequest) err
 	return err
 }
 
+func PullRequestReopen(client *Client, repo ghrepo.Interface, pr PullRequest) error {
```

Comment:
Since other arguments to this function are pointers, perhaps we could accept a pointer to `*PullRequest` as well? With that we could avoid de-referencing `*pr` at call point